pkg/auth/asynccache.go

```diff
@@ -10,6 +10,12 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	initializationRetryInterval = 30 * time.Second
+	initializationTimeout       = 5 * time.Minute
```

Comment:
@deads2k is this ok?

Reply:
@stlaz @deads2k The startupProbe for console pod lasts 5m, which also matches the duration console would retry auth config historically (4.14 and before).